spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -8,14 +8,21 @@ class Klass
   describe '#open_requests' do
     let(:klass) { Klass.new }
 
+
+
     describe 'no unsubmitted feedback' do
-      let(:user) { build(:user) }
-      let(:invitations) { build_list(:invitation, 5) }
+      let(:user) { create(:user) }
+      let(:invitations) { create_list(:invitation, 5, email: user.email) }
 
       it 'should return the count of invitations' do
         Invitation.stub(:where).with(anything()).and_return(invitations)
         klass.open_requests(user).should == 5
       end
+
+      it 'should not include invitations that already have submitted feedback associated with them' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.